.circleci/config.yml

```diff
@@ -118,6 +131,16 @@ jobs:
           command: |
             set -e
             grep -q add code-coverage/coverage.xml
+      - run:
+          name: Verify filtered test results file was created
+          command: |
+            set -e
+            grep -q simpleTest test-results/matlab/filterdtestresult.xml
+      - run:
+          name: Verify filter by tag test results file was created
+          command: |
+            set -e
+            grep -q TaggedTest test-results/matlab/filterdtagresult.xml
```

Comment:
These tests should also confirm that the results **do not** contain a test that should be filtered.

Reply:
Done.